packages/app/src/components/drawer/accounts/switch-accounts.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Box, Flex, Text, Button } from '@stacks/ui';
+import { useWallet } from '@common/hooks/use-wallet';
+import { CheckmarkIcon } from '@components/icons/checkmark-icon';
+import { useRecoilValue, useSetRecoilState } from 'recoil';
+import { accountDrawerStep, AccountStep } from '@store/recoil/drawers';
+import { getAccountDisplayName, getStxAddress } from '@stacks/wallet-sdk';
+import { currentTransactionVersion } from '@store/recoil/networks';
+
+interface SwitchAccountProps {
+  close: () => void;
+}
+export const SwitchAccounts: React.FC<SwitchAccountProps> = ({ close }) => {
+  const { wallet, currentAccountIndex, doSwitchAccount } = useWallet();
+  const setAccountDrawerStep = useSetRecoilState(accountDrawerStep);
+  const transactionVersion = useRecoilValue(currentTransactionVersion);
+  const accountRows = (wallet?.accounts || []).map((account, index) => {
```

Comment:
I'd like to find a nice way to handle the store value being null. Same issues in my wallet, checking for null everywhere.

Reply:
Yeah - I think the ideal might be to have an overload like `useWallet(true)` that will gracefully fail if the user is logged out. And if they're logged in, all the wallet values are non-falsey.